includes/class-wc-payments-fraud-service.php

```diff
@@ -188,4 +207,20 @@ private function get_cookie_session_id() {
 		$cookie_customer_id = $cookie[0];
 		return $wpcom_blog_id . '_' . $cookie_customer_id;
 	}
+
+	/**
+	 * If a "forterToken" cookie is present, send it to the WCPay server so the
+	 * current browsing session can be linked to the account. It will only be sent once.
+	 */
+	public function send_forter_cookie_token() {
+		if ( ! $this->account->is_stripe_connected() || ! isset( $_COOKIE['forterToken'] ) ) {
+			return;
+		}
+
+		$account_id = $this->account->get_stripe_account_id();
+		if ( get_option( 'wcpay_forter_token_sent' ) !== $account_id ) {
+			update_option( 'wcpay_forter_token_sent', $account_id );
+			$this->payments_api_client->send_forter_token( $_COOKIE['forterToken'] );
```

Comment:
What do you think of checking whether `send_forter_token` was successful before setting the option, to prevent an error in the request causing the account never to be tracked?

Reply:
Good idea, done in https://github.com/Automattic/woocommerce-payments/pull/1349/commits/08cdbca9aaf3ff921d8103a93bad142c50adac99